website/docs/docs/cloud/connect-data-platform/connnect-bigquery.md

```diff
@@ -52,6 +52,114 @@ As an end user, if your organization has set up BigQuery OAuth, you can link a p
 
 To learn how to optimize performance with data platform-specific configurations in dbt Cloud, refer to [BigQuery-specific configuration](/reference/resource-configs/bigquery-configs).
 
+### Optional configurations
+
+In BigQuery, optional configurations let you tailor settings for tasks such as running queries, loading or exporting data, and creating tables. While not necessary for basic operations, these options give you greater control over how BigQuery functions behind the scenes to meet your requirements.
+
+To customize your optional configurations in dbt Cloud:
+
+- Select your BigQuery project from the left-hand menu
+- Click **Settings** towards the top right
+- Scroll down to **Development Connection** and select BigQuery
+- Click **Edit** and then scroll down to **Optional settings**
+
+<Lightbox src="/img/bigquery/bigquery-optional-config.png" width="70%" title="BigQuery optional configuration"/>
+
+The following are the optional configs you can set in dbt Cloud:
+
+#### Priority
```

Comment:
based on the google doc - it looks like 'continuous query job' is also supported (in preview) - might be worth confirming with the adapters team if this is supported for dbt and if so, we'd prob need to add it here and in the core doc.